automated install/basic-install.sh

```diff
@@ -167,63 +167,108 @@ is_command() {
     command -v "${check_command}" >/dev/null 2>&1
 }
 
+os_check_dig(){
+    local protocol="$1"
+    local domain="$2"
+    local nameserver="$3"
+    local response
+
+    response="$(dig "${protocol}" +short -t txt "${domain}" "${nameserver}" 2>&1
```

Comment:
Did you forgot the dash before the protocol argument (`-4`, `-6`)?

```suggestion
    response="$(dig -"${protocol}" +short -t txt "${domain}" "${nameserver}" 2>&1
```

Reply:
Thanks for spotting this.